command/server.go

```diff
@@ -549,6 +551,11 @@ func (c *ServerCommand) Run(args []string) int {
 		vault.DefaultMaxRequestDuration = config.DefaultMaxRequestDuration
 	}
 
+	// log proxy settings
+	proxyCfg := httpproxy.FromEnvironment()
```

Comment:
Looks fine. Optionally only log this if any of these values is set, so we don't log if none are set.

Reply:
I settled on unconditional thinking that it would be simpler to be able to grep for "proxy environment" and, if not found, know that the version doesn't emit that message (since it's tough to recall when these minor changes get added).